packages/devextreme/js/events/utils/index.js

```diff
@@ -197,4 +198,6 @@ export const getChar = ({ key, which }) => key || String.fromCharCode(which);
 
 export const addNamespace = mappedAddNamespace;
 
+export const getEventTarget = mappedGetEventTarget;
```

Comment:
[code splitting]
I think it's enough to write your new function directly here, no need to create a separate file.